source/documentation/doc-Release_Notes/chap-Certificates.adoc

```diff
@@ -7,7 +7,7 @@
 
 [WARNING]
 ====
-Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process.
+Do not let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error-prone and time-consuming process.
```

Comment:
I think you should tell them that if the certificates expire, the host and engine stop responding. I think that is better than "the environment becomes non-responsive". 

As for recovery being error-prone and time-consuming -- not sure that is true but we can leave it.